magstock/configspec.ini

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
So far we're just using Uber's config system.  Theoretically we could define our own configuration here, but at the moment we're not doing that because we haven't had any need to do so.